src/sched/job_runner.cc

```diff
@@ -5,6 +5,8 @@
 #include "cris/core/utils/logging.h"
 #include "cris/core/utils/time.h"
 
+#include "nop_asm_impl.h"
```

Comment:
path?

Reply:
This is not a part of "header", so it is not copied to the virtual include path directory. We treat it as src file so that it won't be exposed externally, so it is only visible in the "current directory".

Reply:
Header should always have full path, or the build result would depend on this kind of tiny details specific to the build system, and that's really bad.

Reply:
Header is just header, it doesn't have any concept of public or private. What you describe should be part of the packaging process, not build.

Reply:
> Header is just header, it doesn't have any concept of public or private. 

C++ does not, but Bazel does. Bazel treat headers and sources differently. The headers will be copied to "virtual include paths" so that other source files only sees files that is treated as "headers" when `#include` with full path. "Sources" won't be in those virtual include paths so if you use the "full path", that is how we make it "private".

Reply:
Remove the whole "private header" thing anyways because we don't need it here.